polkadot/runtime/westend/src/impls.rs

```diff
@@ -157,7 +157,11 @@ where
 					.into(),
 			},
 			// Poke the deposit to reserve the appropriate amount on the parachain.
-			Transact { origin_kind: OriginKind::Superuser, call: poke.encode().into() },
+			Transact {
+				origin_kind: OriginKind::Superuser,
+				call: poke.encode().into(),
+				fallback_max_weight: None,
```

Comment:
isn't this `ToParachainIdentityReaper` obsolete now? used for identity migration which has since completed? maybe just remove it in this or separate PR.

if still used, then this should be `Some(fallback_weight)` - Relay will upgrade first to v5 and People chain will still be on v4 for a bit before upgrading.

Reply:
Added it back